test/kb_blast_server_test.py

```diff
@@ -126,24 +249,128 @@ def test_kb_blast_BLASTn_Search_01(self):
         self.assertIsNotNone(report_obj['objects_created'][0]['ref'])
 
         created_obj_0_info = self.getWsClient().get_object_info_new({'objects':[{'ref':report_obj['objects_created'][0]['ref']}]})[0]
+        self.assertEqual(created_obj_0_info[NAME_I], obj_out_name)
+        self.assertEqual(created_obj_0_info[TYPE_I].split('-')[0], obj_out_type)
+
+        # check number of hits in featureSet output
+        featureSet_out_obj = self.getWsClient().get_objects([{'ref':report_obj['objects_created'][0]['ref']}])[0]['data']
+        self.assertEqual(expected_hit_cnt, len(featureSet_out_obj['element_ordering']))
+        pass
+
+
+    # Test BLASTn: GenomeSet target
+    #
+    # Uncomment to skip this test
+    # HIDE @unittest.skip("skipped test_kb_blast_BLASTn_Search_01")
+    def test_kb_blast_BLASTn_Search_01(self):
         [OBJID_I, NAME_I, TYPE_I, SAVE_DATE_I, VERSION_I, SAVED_BY_I, WSID_I, WORKSPACE_I, CHSUM_I, SIZE_I, META_I] = list(range(11))  # object_info tuple
-        self.assertEqual(created_obj_0_info[NAME_I], target_1+'-'+obj_out_name)
+
+        obj_basename = 'BLASTn'
+        obj_out_name = obj_basename+".test_output.FS"
+        obj_out_type = "KBaseCollections.FeatureSet"
+        genomeSet_name = 'test_genomeSet.BLASTn.GenomeSet'
+        expected_hit_cnt = 3
+        
+        genomeInfo_0 = self.getGenomeInfo('GCF_001566335.1_ASM156633v1_genomic', 0)  # E. coli K-12 MG1655
+        genomeInfo_1 = self.getGenomeInfo('GCF_000021385.1_ASM2138v1_genomic', 1)    # D. vulgaris str. 'Miyazaki F
+        genomeInfo_2 = self.getGenomeInfo('GCF_001721825.1_ASM172182v1_genomic', 2)  # Pseudomonas aeruginosa
+        genomeInfo_3 = self.getGenomeInfo('GCF_002950035.1_ASM295003v1_genomic', 3)  # Shigella boydii
+        genomeInfo_4 = self.getGenomeInfo('GCF_000512125.1_ASM51212v1_genomic', 4)  # Escherichia albertii KF1
+        genome_ref_1 = self.get_obj_ref_from_obj_info(genomeInfo_0)
+        genome_ref_2 = self.get_obj_ref_from_obj_info(genomeInfo_1)
+        genome_ref_3 = self.get_obj_ref_from_obj_info(genomeInfo_2)
+        genome_ref_4 = self.get_obj_ref_from_obj_info(genomeInfo_3)
+        genome_ref_5 = self.get_obj_ref_from_obj_info(genomeInfo_4)
```

Comment:
Rather than making a whole load of similarly named variables, it'd be better to create a couple of lists or a dictionary with this info in it.

e.g.
```py
input_genomes = [
  { # E. coli K-12 MG1655
    "file": 'GCF_001566335.1_ASM156633v1_genomic',
    "sciname": "FOO",
  },
  { # D. vulgaris str. 'Miyazaki F'
    "file": "GCF_000021385.1_ASM2138v1_genomic",
    "sciname": "BAR",
  },
  ...
]
for genome in input_genomes:
  genome["info"] = self.getGenomeInfo(genome["file"])
  genome["ref"] = self.get_obj_ref_from_obj_info(genome["info"])

genome_ref_list = [genome["ref"] for genome in input_genomes]
genome_scinames = [genome["sciname"] for genome in input_genomes]
```

Reply:
done